client/src/app/project-create-edit/schemas/project-metadata-schema.json

```diff
@@ -451,6 +452,12 @@
             "type": "string",
             "user_friendly": "Publication URL",
             "example": "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5667944/"
+          },
+          "official_hca_publication": {
+            "description": "Has the publication been accepted as an official HCA publication, according to the process described in https://www.humancellatlas.org/publications/ ?",
+            "type": "boolean",
+            "user_friendly": "Official HCA Publication",
+            "example": "yes; no"
```

Comment:
I realised you didn't put the `guidelines`. Is that intended so that the helper text on the right side of the field will not show `Should be one of: yes, or no.` ?

Reply:
Yes exactly!

Reply:
I see, yeah it makes sense, thanks!

I think the guidelines field is intended for guidelines when filling out the spreadsheets 🤔 it seems not appropriate for the UI forms.